src/Tribe/Timezones.php

```diff
@@ -66,22 +77,29 @@ public static function wp_timezone_abbr( $date ) {
 	 * @return string
 	 */
 	public static function wp_timezone_string() {
+		static $cache_key = __METHOD__;
+		$cached_value = tribe_get_var( $cache_key );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 5 spaces but found 1 space

<sub>hash: 0285facfa9fc32bee2d5b851b477e316</sub>